run/awake_start

```diff
@@ -78,13 +84,23 @@ case "${TESLA_WAKE_MODE:-stream}" in
     is_sentry_mode_enabled=$(/root/bin/tesla_api.py is_sentry_mode_enabled | tr '[:upper:]' '[:lower:]')
     if [ "false" = "${is_sentry_mode_enabled}" ]
     then
-      log "Temporarily enabling Sentry Mode to keep car awake."
-      touch /tmp/disable_sentry_after_archiving
-      /root/bin/tesla_api.py enable_sentry_mode &>> "${LOG_FILE}"
+      # Check if TESLAFI_WAKE_MODE is set to sentry
+      if [ "$TESLAFI_WAKE_MODE" == "sentry" ]; then
+        log "Turning on Sentry Mode to keep car awake (via TeslaFI)."
+        set_sentry_mode true
```

Comment:
This will only be called if TESLA_WAKE_MODE is also set to "sentry", and the condition on line 85 is true, which makes no sense because:
- it will always be true because the API that that uses no longer works
- even if it did, it would then require configuring the Tesla API variables in the config file, and the config specifically says to only use one of the APIs

This new TeslaFi code should be separate, and should use the TeslaFi API to determine whether sentry mode is already on.